### ISSUE TITLE
Append information about failed soy parsing

### ISSUE THREAD

**Issue by jbalsas** [metal/metal-tools-soy#16]
Currently, we're simply emitting an error when a soy parsing fails, but we're not saying where did it fail in some cases. We should always append that information. For example:

```
Running 'build'...
Compiling soy
Error in plugin 'metal-tools-soy'
Message:
    Parsing failed at 5:4. Expecting:
' '
'@param'
a character matching function (ch) {
      return str.indexOf(ch) < 0;
    }
```

**Comment by robframpton** [comment]
Ran into this today, it would be very helpful to provide the file path where the error occurred.

**Comment by jbalsas** [comment]
Fixed in #20! 🎉
